examples/cpp/rtps_entities/README.md

```diff
@@ -0,0 +1,98 @@
+# RTPS entities example
+
+The *eProsima Fast DDS RTPS entities* example is a simple application intended to demonstrate a basic RTPS deployment.
+
+This example is part of the suite of examples designed by eProsima that aims to illustrate the features and possible configurations of DDS and RTPS deployments through *eProsima Fast DDS*.
+
+In this case, the *RTPS entities* example describes the simplest deployment of a Fast DDS reader and writer.
+
+* [Description of the example](#description-of-the-example)
+* [Run the example](#run-the-example)
+
+## Description of the example
+
+Each example application creates a different entity in the RTPS (Real-Time Publish-Subscribe) layer: writer and reader;
+The lower-level RTPS layer in eProsima Fast DDS implements the RTPS protocol as defined in the standard.
+This layer offers more granular control over the communication protocol's internals compared to the higher-level DDS (Data Distribution Service) Layer, giving advanced users greater command over the library's features.
+
+The serialization of the payload is done on the writer side and is designed to convert a `HelloWorld` object into a serialized data format.
+This is useful for transmitting the object over a network or saving it in a compact form for later use.
+The deserialization of the payload on the reader side consists of converting a serialized data payload into a usable `HelloWorld` object.
+This is typically used in applications where data is transmitted in a compact format and needs to be deserialized back into its original form for further processing.
+
+
+## Run the example
+
+To launch this example, two different terminals are required.
+One of them will run the reader example application, and the other will run the writer application.
+
+### RTPS entities writer
+
+* Ubuntu ( / MacOS )
+
+    ```shell
+    user@machine:example_path$ ./rtps_entities writer
+    RTPS Writer running. Please press Ctrl+C to stop the RTPS Writer at any time.
+    ```
```

Comment:
(apply to all)
```suggestion
### RTPS writer

* Ubuntu ( / MacOS )

    ```shell
    user@machine:example_path$ ./rtps writer
    RTPS Writer running. Please press Ctrl+C to stop the RTPS Writer at any time.
    ```
```